compiler/luci/pass/src/helpers/ExpressionCache.cpp

```diff
@@ -118,8 +118,17 @@ Expression Expression::build(luci::CircleNode *node)
 
   Expression key;
   {
-    for (uint32_t i = 0; i < node->arity(); i++)
-      key.inputs.emplace_back(node->arg(i));
```

Comment:
Previous code considers all inputs including constants. This has to be updated to consider only input feature maps, not all inputs.